src/Logger/BadgeLogger.php

```diff
@@ -66,26 +67,60 @@ public function log()
             return;
         }
 
-        $apiKey = getenv(self::ENV_INFECTION_BADGE_API_KEY);
         $repositorySlug = getenv('TRAVIS_REPO_SLUG');
         $currentBranch = getenv('TRAVIS_BRANCH');
 
-        if ($apiKey && $repositorySlug && $currentBranch === $this->config->branch) {
-            $this->badgeApiClient->sendReport(
-                $apiKey,
-                'github.com/' . $repositorySlug,
-                $currentBranch,
-                $this->metricsCalculator->getMutationScoreIndicator()
-            );
-        } else {
+        if ($repositorySlug === false || $currentBranch === false) {
+            $this->showInfo('repository slug nor current branch were found; not a Travis build?');
+
+            return;
+        }
+
+        if ($currentBranch !== $this->config->branch) {
             $this->showInfo(
                 sprintf(
-                    'Repository Slug=%s, Branch=%s',
-                    $repositorySlug,
+                    'expected branch "%s", found "%s"',
+                    $this->config->branch,
                     $currentBranch
                 )
             );
+
+            return;
         }
+
+        $apiKey = getenv(self::ENV_INFECTION_BADGE_API_KEY);
+
+        /*
+         * Original Stryker Dashboard manual warrants a different API key:
+         * fall back to theirs if our isn't present
+         */
+        if ($apiKey === false) {
+            $apiKey = getenv(self::ENV_STRYKER_DASHBOARD_API_KEY);
```

Comment:
In this case, please update the documentation. Now it says:

`Make sure to use INFECTION_BADGE_API_KEY environment variable name instead of STRYKER_DASHBOARD_API_KEY.`

Both should be allowed.

Reply:
Done: https://github.com/infection/site/pull/49